src/scene/materials/options.js

```diff
@@ -0,0 +1,7 @@
+class Options {
```

Comment:
we need to give it a less generic name than `Options`. Maybe `StandardOptions`?

Reply:
Done